FurCoNZ/Views/Order/EditorTemplates/TicketDetailViewModel.cshtml

```diff
@@ -0,0 +1,93 @@
+                @Model.TicketTypeName - Ticket #@Model.Id
+            </button>
+        </h2>
+    </div>
+
+    <div id="attendee-@Model.Id-detail" class="collapse" aria-labelledby="attendee-@Model.Id-header" data-parent="#attendee-details-accordion">
+        <div class="card-body">
+            <!-- #region Attendee Details Form -->
+
+            <input asp-for="Id" type="hidden" />
+            <input asp-for="TicketTypeId" type="hidden" />
+
+            <div class="form-group">
+                <label asp-for="EmailAddress">Email address</label>
+                <input type="email" class="form-control" asp-for="EmailAddress" placeholder="Enter email" />
+            </div>
+
+            <ul class="nav nav-pills mb-3" id="attendee-@Model.Id-pills-tab" role="tablist">
+                <li class="nav-item">
+                    <a class="nav-link" id="attendee-@Model.Id-pills-self-tab" data-toggle="pill" href="#attendee-@Model.Id-pills-self" role="tab" aria-controls="attendee-@Model.Id-pills-self" aria-selected="false">This is my ticket, or I am filling this out for someone else</a>
+                </li>
+                <li class="nav-item">
+                    <a class="nav-link" id="attendee-@Model.Id-pills-share-tab" data-toggle="pill" href="#attendee-@Model.Id-pills-share" role="tab" aria-controls="attendee-@Model.Id-pills-share" aria-selected="false">This is for someone else</a>
+                </li>
+            </ul>
+
+            <div class="tab-content" id="attendee-@Model.Id-pills-tabContent">
+                <div class="tab-pane fade" id="attendee-@Model.Id-pills-self" role="tabpanel" aria-labelledby="attendee-@Model.Id-pills-self-tab">
+                    <div class="form-group">
+                        <label asp-for="BadgeName">Badge Name</label>
+                        <input class="form-control" asp-for="BadgeName" placeholder="Enter email">
+                    </div>
+                    <div class="form-group">
+                        <label asp-for="PreferredFullName">Full Name</label>
+                        <input asp-for="PreferredFullName" class="form-control" />
+                    </div>
+                    <div class="form-group">
+                        <label asp-for="PreferredKnownAs">What should we call you, for short?</label>
+                        <input asp-for="PreferredKnownAs" class="form-control" aria-describedby="attendee-@Model.Id-preferredNameHelp" />
+                        <small id="attendee-@Model.Id-preferredNameHelp" class="form-text text-muted">(usually your first name, e.g. "Matt" or "Sarah")</small>
+                    </div>
+
+                    <div class="form-check">
+                        <input class="form-check-input" type="checkbox" value="" id="attendee-@Model.Id-isSameAsId">

```

Comment:
This should really be underneath *Full Name*. Being detached from `IdentificationFullName`. 
Naturally I interpreted "What should we call you for short?" as the subject for the question "This is the name on my identification"

Reply:
I've removed "What should we call you for short?", it's largely more confusing than helpful and wasn't actually asked for.